packages/api/es/queries.js

```diff
@@ -65,9 +61,7 @@ const build = {
     }
 
     // check for boolean values
-    if (typeof i.value === 'number') {
```

Comment:
The lodash `isNumber` function handles this a little more completely: `typeof value == 'number' || (isObjectLike(value) && baseGetTag(value) == numberTag)`